keyboards/typek/info.json

```diff
@@ -0,0 +1,110 @@
+{
+    "keyboard_name": "TypeK-S Rev. Zeta-RC2",
+    "manufacturer": "gok",
+    "usb": {
+        "pid": "0x8475",
+        "vid": "0x7179",
+        "device_version": "6.2.0"
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["A3" , "B12", "A0" , "A10", "B0" , "B1" , "B10", "B3" , "C15", "C14", "C13", "B9" , "B8" , "B5" ],
+        "rows": ["A7" , "A8" , "A15", "A1" , "A2" ]
+    },
+    "features": {
+        "bootmagic": true,
+        "lto":       true,
+        "mousekey":  false,
+        "extrakey":  false,
+        "console":   false,
+        "backlight": false,
+        "rgblight":  true,
+        "audio":     false,
+        "encoder":   false,
+        "nkro":      true
+    },
+    "ws2812":{
+        "pin": "B15"
+    },
+    "rgblight": {
+        "led_count": 69
+    }, 
+    "url": "https://www.gok.design/type-k",
+    "maintainer": "Gondolindrim", 
+    "processor": "STM32F411",
+    "bootloader": "stm32-dfu",
+    "layouts": {
```

Comment:
Could a layout be added without split bars and split keys?
[reference image](https://i.imgur.com/lGLFYnZ.png)

Reply:
Since there are four choices (two split spacebars, split right shift and split backspace) this would total 16 different layouts and I would really rather not do it...

Reply:
To help rephrase the same question:
Could one additional layout be added, to `layouts` where there are no split keys nor split spacebars?

Reply:
Added in [d42162a](https://github.com/qmk/qmk_firmware/pull/22876/commits/d42162a5006a00d066832cdb0bfdd0b8fb2ef8da)